crates/astria-core/src/protocol/transaction/v1alpha1/action.rs

```diff
@@ -169,6 +169,14 @@ impl Action {
         };
         Some(transfer_action)
     }
+
+    pub fn is_fee_asset_change(&self) -> bool {
```

Comment:
Let's make a tracking issue to add unit tests for these.

Reply:
created and commented in code: https://github.com/astriaorg/astria/issues/1593